ObjectiveC/Ambly Demo/Ambly Demo.xcodeproj/project.pbxproj

```diff
@@ -154,17 +193,28 @@
 			projectRoot = "";
 			targets = (
 				ED387D181A602B56007E40A2 /* Ambly Demo */,
+				AA6422291C28A5D7002F7170 /* Ambly Demo TV */,
 			);
 		};
 /* End PBXProject section */
 
 /* Begin PBXResourcesBuildPhase section */
+		AA6422311C28A5D7002F7170 /* Resources */ = {
+			isa = PBXResourcesBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+				AA25B1891C2B5567002F61FD /* Main-TV.storyboard in Resources */,
+				AA6422341C28A5D7002F7170 /* Images.xcassets in Resources */,
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+		};
 		ED387D171A602B56007E40A2 /* Resources */ = {
 			isa = PBXResourcesBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
 				ED387D281A602B56007E40A2 /* Main.storyboard in Resources */,
 				ED387D2D1A602B56007E40A2 /* LaunchScreen.xib in Resources */,
+				AA64223D1C28A8E5002F7170 /* Info-TV.plist in Resources */,
```

Comment:
`Info-TV.plist` is listed under Build Phases > Copy Bundle Resources for the Ambly Demo target (the non-tvOS target). I bet it causes no issue but is also probably caused by this line here.